app/src/fdroid/java/com/eventyay/organizer/utils/BrowserUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.eventyay.organizer.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+
+public final class BrowserUtils {
+
+    private BrowserUtils() {
+    }
+
+    public static void launchUrl(String url, Context context) {
```

Comment:
`context, url`

Reply:
Is it a standard to write `Context` first?

Reply:
Yes

Reply:
When creating static methods, we first define the state, the non changing variables and then the actual argument. The position of the argument should define its variability in ascending order. That's why optional arguments or settings are last. They have most variability, they can not exist as well